storagenode/pieces/cache.go

```diff
@@ -33,12 +34,13 @@ type CacheService struct {
 
 // NewService creates a new cache service that updates the space usage cache on startup and syncs the cache values to
 // persistent storage on an interval.
-func NewService(log *zap.Logger, usageCache *BlobsUsageCache, pieces *Store, interval time.Duration) *CacheService {
+func NewService(log *zap.Logger, usageCache *BlobsUsageCache, pieces *Store, interval time.Duration, PieceScanOnStartup bool) *CacheService {
```

Comment:
Still failing lint: `PieceScanOnStartup` should not be capitalised as a function parameter

Reply:
Fixed! Sorry for that.

The CI keeps failing but I’m not able to know which test fails 😅

Reply:
Yeah the build failures are not related to your change. We'll have a look

Reply:
Fixed in https://github.com/storj/storj/pull/5024/commits/4908eda1137fb8547c4cf52b925f208dff9cf19f !